subprojects/gradle/runner/client/src/main/kotlin/com/avito/runner/scheduler/metrics/TestMetricsAggregator.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.runner.scheduler.metrics
+
+internal interface TestMetricsAggregator {
+
+    fun initialDelay(): Long?
```

Comment:
Why they could be nullable?

Reply:
could be a lot of reasons:
- something it broken in runner logic and some listener calls not invoked
- empty test suite
- maybe some case in integration tests

It will be very hard to troubleshoot with error message from metrics sender, so i prefer defensive approach here, but log a warning that metrics was not sended

Reply:
ok